### ISSUE TITLE
Exported save file is inaccessable

### ISSUE THREAD

**Issue by eco9898** [kwsch/PKHeX.Mobile#33]
Due to the latest version of Android, (android 11), android/data is no longer an easy access folder. Can I suggest changing the export function to just access a pkhex folder in the /storage/emulated/0/ folder instead. This would allow users to access the save file they export

**Comment by kwsch** [comment]
It already does that on the latest commit, and pipeline build.

**Comment by eco9898** [comment]
Sorry, I thought I pulled the latest last night. Maybe I'm doing something wrong when I build causing it to use old code.

**Comment by eco9898** [comment]
I'm going to try manually upgrading it for android 11 myself since it does appear to use the android 10 file permission system still. I'm just following this but probably gonna mess something up: https://stackoverflow.com/questions/31746787/xamarin-android-system-unauthorizedaccessexception-access-to-the-path-is-de

I haven't worked on android development before so I apologize if I have overlooked something.

**Comment by eco9898** [comment]
I modified it to create a PkHex folder and to place the exported save into it. It uses android 11 permission: MANAGE_EXTERNAL_STORAGE
I wasn't sure how to implement:
`private void RequestStorageAccess()
{
    if (!Environment.IsExternalStorageManager) 
    {
        StartActivityForResult(new Intent(Android.Provider.Settings.ActionManageAllFilesAccessPermission), 3);
    }
}`
so I set up an error message promting the user to enable the permission if it failes.
By manually allowing the permission in the settings it works, although I have been told this is bad practice and there is a new file system API but I don't know how to go that step further.
As this application is currently build and deploy yourself, rather than using the play store, I don't see the issues with using this permission for now.

I did try to set it up to create a sub folder with the date and time, but for some reason saving the file to this folder caused permission errors again
